pkg/core/checker.go

```diff
@@ -22,3 +22,30 @@ func (NoopChecker) Check(m Model, ops []Operation) (bool, error) {
 func (NoopChecker) Name() string {
 	return "NoopChecker"
 }
+
+type multiChecker struct {
+	checkers []Checker
```

Comment:
Maybe a `build(...args)` method or append is better...

Reply:
`MultiChecker` is the build method

Reply:
Got